src/client/containers/Home/Home.js

```diff
@@ -42,20 +42,25 @@ export const Home = () => {
         </div>
         <div className="home-button">
           <div className="home-button-right">
-            <Buttons
-              label="Take the career quiz"
-              size="big"
-              isMono={true}
-              color="orange"
-            />
+            <Link to="/CareerPage">
+              <Buttons
+                label="Take the career quiz"
+                size="big"
+                isMono={true}
+                color="orange"
```

Comment:
This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an `<a>` tag around a `<button>` tag. 

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Reply:
If you want to navigate to a page using a button this is the correct way to do so :). 

https://reactrouter.com/web/api/Hooks/usehistory

Reply:
we changed